src/components/TransactionList/TransactionListTs.ts

```diff
@@ -207,21 +180,18 @@ export class TransactionListTs extends Vue {
 
   /**
    * Returns all the transactions,
-   * from the getter that matches the provided selected option
+   * from the getter that matches the provided tab name
+   * @param {TabName} group
+   * @returns {Transaction[]}
    */
-  public getTransactionsByStatus(status: string): Transaction[] {
-    switch (status) {
-      case 'confirmed':
-        return this.confirmedTransactions
-      case 'unconfirmed':
-        return this.unconfirmedTransactions
-      case 'partial':
-        return this.partialTransactions
-      case 'all':
-        return [ ...this.confirmedTransactions,...this.partialTransactions,...this.unconfirmedTransactions ]
-      default:
-        return []
-    }
+  public getCurrentTabTransactions(group: TransactionGroup): Transaction[] {
+    if (group === TransactionGroup.confirmed) return this.confirmedTransactions
+    if (group === TransactionGroup.unconfirmed) return this.unconfirmedTransactions
+    if (group === TransactionGroup.partial) return this.partialTransactions
+    if (group === TransactionGroup.all) {return [ ...this.confirmedTransactions,
+      ...this.partialTransactions, ...this.unconfirmedTransactions ]}
+
+    return []
   }
```

Comment:
maybe  the `switch/case` is better，my personal point.

Reply:
this code does not respect linting.